frontend/entrypoint.sh

```
@@ -0,0 +1,38 @@
+#!/bin/sh
+set -e
+
+echo "Loading cmsch frontend"
+echo ${VITE_API_BASE_URL}
+
+NGINX_HTML=/usr/share/nginx/html
+
+# Substitute HTML-level env vars into index.html (icons, fonts, manifest, error reporter, Plausible)
+envsubst '${VITE_API_BASE_URL} ${VITE_NAME} ${VITE_DESCRIPTION} ${VITE_THEME_COLOR} ${VITE_PLAUSIBLE_URL}' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)